tests/src/test/scala/it/unibo/scafi/test/functional/TestScafiTestUtils.scala

```diff
@@ -0,0 +1,74 @@
+package it.unibo.scafi.test.functional
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import org.scalatest._
+
+class TestScafiTestUtils extends FlatSpec {
+  import ScafiTestUtils._
+
+  val ManhattanNet = new ItWord
+
+  ManhattanNet should "Support every possible detachment" in {
+    val net: Network with SimulatorOps = manhattanNet(
+      detachedNodesCoords = Set((0,0), (2,0), (0,2), (2,2))
+    )
+
+    assert(net.neighbourhood(0).isEmpty)
+    assert(net.neighbourhood(1) == Set(3,4,5))
+    assert(net.neighbourhood(2).isEmpty)
+    assert(net.neighbourhood(3) == Set(1,4,7))
+    assert(net.neighbourhood(4) == Set(1,3,5,7))
+    assert(net.neighbourhood(5) == Set(1,4,7))
+    assert(net.neighbourhood(6).isEmpty)
+    assert(net.neighbourhood(7) == Set(3,4,5))
+    assert(net.neighbourhood(8).isEmpty)
```

Comment:
I do not remember. There was a reason for which in this and the following testcases we do not use a single assertion "as usual" where the assertion value is structured using a "grid" shape through line breaks?

Reply:
In this case I'm testing a property of the network, the helper function `assertNetworkValues` checks against node's value from the export. 

Using 
```scala

    assertNetworkValues((0 to 8).zip(List(
      net.neighbourhood(0).isEmpty, net.neighbourhood(1) == Set(3,4,5), net.neighbourhood(2).isEmpty,
      net.neighbourhood(3) == Set(3,4,5),  net.neighbourhood(4) == Set(3,4,5),  net.neighbourhood(5) == Set(3,4,5),
      net.neighbourhood(6).isEmpty, net.neighbourhood(7) == Set(3,4,5), net.neighbourhood(8).isEmpty
    )).toMap)(net)
```

Will result in the following netwok:
```
Actual network: 
_	_	_	
_	_	_	
_	_	_
```